Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc

```diff
@@ -153,6 +161,11 @@ export function runQTest(args: QTestArguments): Result {
             "--qTestRawArgFile ",
             Artifact.input(args.qTestRawArgFile)
         ),
+        Cmd.option(
+            "--qCodeCoverageEnumType ",
+            qCodeCoverageEnumTypeToString(args.qCodeCoverageEnumType)

```

Comment:
See below for more details why:

Option 1: If you don't always want to specify the argument, only when users define it.
```
Cmd.option("--qCodeCoverageEnumType ", args.qCodeCoverageEnumType)
```
Option 2: If you always want to see it on the ocmmandline but select "None" when not specified.
```
Cmd.option("--qCodeCoverageEnumType ", args.qCodeCoverageEnumType || "None")
```